### PR TITLE
Update chrf.py to remove torch Warnings

### PR DESCRIPTION
**Pull request by Shoumik-Gandre**
## What does this PR do?
Previously, line 192 was causing the following warning:
```
 UserWarning: To copy construct from a tensor, it is recommended to use sourceTensor.clone().detach() or sourceTensor.clone().detach().requires_grad_(True), rather than torch.tensor(sourceTensor).
  total_n_grams[n] = tensor(sum(n_grams_counts[n].values()))
```

My changes solves this warning.

Fixes #\<issue_number>

<details>
  <summary>Before submitting</summary>

- [ ] Was this **discussed/agreed** via a Github issue? (no need for typos and docs improvements)
- [x] Did you read the [contributor guideline](https://github.com/Lightning-AI/torchmetrics/blob/master/.github/CONTRIBUTING.md), Pull Request section?
- [ ] Did you make sure to **update the docs**?
- [ ] Did you write any new **necessary tests**?

</details>

<details>
  <summary>PR review</summary>

Anyone in the community is free to review the PR once the tests have passed.
If we didn't discuss your PR in Github issues there's a high chance it will not be merged.

</details>

## Did you have fun?
Yes, this is my first-ever pull request. I encountered this warning repeatedly (super annoying) and it had a very easy solution to it.

Make sure you had fun coding 🙃

----
📚 Documentation preview 📚: https://torchmetrics--2482.org.readthedocs.build/en/2482/

### CODE REVIEW

**Review by Borda — APPROVED** [review]

**Review by SkafteNicki — APPROVED** [review]

**Review by stancld — APPROVED** [review]

**Comment by SkafteNicki** [comment]
It is very strange to me that this simple change to `chrf` metric results in the `BLEUScore` metric consistently falling in some CI builds. To my knowledge the two metrics are completely independent of each others.
@stancld do you have a idea what is going on?
